scripts/tools-versions-linux.yml

```diff
@@ -3,7 +3,7 @@ python:
 git:
   version: 2.37.3
 cmake:
-  version: 3.20.6
```

Comment:
can you change `tools-versions-darwin.yml` also? 
https://github.com/nrfconnect/sdk-nrf/blob/main/scripts/tools-versions-darwin.yml#L6